net-irc/inspircd/inspircd-2.0.26.ebuild

```diff
@@ -0,0 +1,123 @@
+# Copyright 1999-2018 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI="6"
+
+inherit toolchain-funcs user versionator
```

Comment:
Please use eapi7-ver instead of versionator.

Reply:
Done.